precompile/contracts/warp/config.go

```diff
@@ -202,7 +208,7 @@ func (c *Config) VerifyPredicate(predicateContext *precompileconfig.PredicateCon
 		context.Background(),
 		&warpMsg.UnsignedMessage,
 		predicateContext.SnowCtx.NetworkID,
-		warpValidators.NewState(predicateContext.SnowCtx), // Wrap validators.State on the chain snow context to special case the Primary Network
+		warpValidators.NewState(predicateContext.SnowCtx, func() bool { return c.RequirePrimaryNetworkSigners }), // Wrap validators.State on the chain snow context to special case the Primary Network
```

Comment:
I think this is fine since we don't call `VerifyPredicate` in bootstrapping, right? Otherwise this might break predicate verification in older blocks. 

hmm or is it fine because in order to activate this config one needs to disable and then re-enable warp with this config?

Reply:
this config should always correspond to the active config of the precompile (so seems similar to the 2nd description)